### ISSUE TITLE
osx: build depending on tbb 2019.1-h2d50403_0 fails

### ISSUE THREAD

**Issue by looooo** [conda-forge/tbb-feedstock#42]
freecad fails again with the latest tbb-package:

tbb:             2019.1-h2d50403_0       conda-forge
https://circleci.com/gh/looooo/freecad-feedstock/51?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
```
.../include/tbb/task_group.h:132:53: error: 'uncaught_exceptions' is unavailable: introduced in macOS 10.12
```

This was working with the previous available version of tbb:

tbb:             2019.0-h2d50403_0       conda-forge
https://circleci.com/gh/looooo/freecad-feedstock/48?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by anton-malakhov** [comment]
I recommend submitting this issue against tbb's source repository: https://github.com/01org/tbb

**Comment by looooo** [comment]
The only thing I changed was the build-number. But it's possible other (newer) packages were pulled. I will have a look...

**Comment by anton-malakhov** [comment]
@looooo BTW, the gates for U2 are closing now.. so, please, hurry up with submitting the issue against tbb for this

**Comment by akukanov** [comment]
Note that in TBB 2018 and higher we do not support OS X 10.9, and it is not tested anymore. We will fix the issue this time, but I recommend you to either switch to 10.11 or stay at an older version of TBB.

**Comment by anton-malakhov** [comment]
Great, thanks Alexey!

**Comment by looooo** [comment]
thanks for having a look at this.

> but I recommend you to either switch to 10.11 or stay at an older version of TBB.

this means we should pin the next version of tbb within conda-forge-pinning:
https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/123

**Comment by anton-malakhov** [comment]
hm.. that's basically raises the question that simple tests and examples are not enough for testing the packages for compatibility with conda-forge environment. :( I'll need to come up with some way of bringing a subset of TBB's unittests to the conda test stage

**Comment by anton-malakhov** [comment]
@looooo Please check if 2019.2 fixes the OS X 10.9 target

**Comment by looooo** [comment]
works for osx-builds of freecad. The latest build used this version: tbb 2019.2. Thanks for the support.
